src/content/docs/config/gated-content.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Gated content
+description: Learn how to implement gated content for assets that are not managed by Adobe Commerce.
+---
+
+import Diagram from '@components/Diagram.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+import Callouts from '@components/Callouts.astro';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+import Aside from '@components/Aside.astro';
+
+You might want to restrict access to certain content, such as videos, images, or pages on your site that is not managed by Adobe Commerce. This content resides on the edge and cannot be controlled using customer segments or other Commerce functionality. You can restrict access to these non-Commerce assets using gated content. 
+
+## Key concepts
```

Comment:
For the sake of consistency, I recommend using the "vocabulary" pattern using the `<Vacabulary>` component: https://github.com/commerce-docs/microsite-commerce-storefront/blob/develop/src/content/docs/get-started/index.mdx?plain=1#L64C1-L86